packages/grafana-data/src/types/dataFrameTypes.ts

```diff
@@ -7,7 +7,7 @@
 export enum DataFrameType {
   TimeSeriesWide = 'timeseries-wide',
   TimeSeriesLong = 'timeseries-long',
-  TimeSeriesMany = 'timeseries-many',
```

Comment:
Is this a breaking change in our public (typescript) API?

Reply:
This is a good question! @kylebrandt @ryantxu Is this supposed to be a breaking change or not? We need to consider the typescript API as Josh mentioned.

Reply:
what is and isn't a breaking changes for plugins is tricky to define for the frontend. Typescript is not a runtime concern, so this will not break any existing plugins from running. But it could prevent them from being built.

From the outside, a simple way out would be to actually _deprecate_ (rather than remove) the enum value with `@deprecated` jsdoc annotation, and then remove in grafana 10.

Reply:
Exactly what @joshhunt said - https://github.com/grafana/grafana/blob/main/contribute/breaking-changes-guide.md#deprecate-first

Reply:
Yup deprecating not removing

Reply:
Can we add something to point to the replacement, e.g. "use TimeSeriesMulti instead", or "@deprecated in favor of TimeSeriesMulti"